### PR TITLE
Add sandbox gamemode effects

### PR DESCRIPTION
**Pull request by ReasonableScripts**
the last few effects from the bottom of this page https://wiki.facepunch.com/gmod/Effects

### CODE REVIEW

File: lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua [hunk 1]
```diff
@@ -66,5 +66,16 @@ GmodScripts_EffectWhitelist = {
     acf_heat_explosion = true,
     simfphys_engine_smoke = true,
     eff_jack_plasmaarc = true,
+
+    balloon_pop = true,
```
Comment (wrefgtzweve) [inline lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua:70]: `balloon_pop` can be abused to cause severe lag

Comment (ReasonableScripts) [reply lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua:70]: is there a way to remove it from the pr then?

Comment (plally) [reply lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua:70]: edit  your branch to remove it

Comment (plally) [reply lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua:70]: if youre using the web editor you can do it here
https://github.com/ReasonableScripts/cfc_gmod_scripts/blob/patch-1/lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua

**Comment by wrefgtzweve** [comment]
i think some of the other effects can be bad too but i don't recognize them directly by effect name, so i suggest someone tests this before its merged

File: lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua (hunk 1 above)
Comment (ReasonableScripts) [reply lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua]: did that fix it?

**Comment by ReasonableScripts** [comment]
i just need to know what to remove :)

**Review by brandonsturgeon — APPROVED** [review]
Apologies for the late response on this, it's been a busy year for us!

I'll approve this, and if we find that any of these effects are being abused we'll remove them.

Thank you for the contribution 👍

File: lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua [hunk 2]
```diff
@@ -71,5 +71,15 @@ GmodScripts_EffectWhitelist = {
     acf_heat_explosion = true,
     simfphys_engine_smoke = true,
     eff_jack_plasmaarc = true,
+
```
Comment (brandonsturgeon) [inline lua/cfc_gmod_scripts/effects_whitelist/sh_init.lua:74]: Quick spacing fix 👍 
```suggestion

```

**Pull request merged**